cloud-config/configs/elasticsearch/es-cluster-elect.yml

```diff
@@ -9,10 +9,11 @@
 network.host: 0.0.0.0
 http.port: 9201
 transport.tcp.port: 9299
-discovery.type: ec2
-cloud.aws.region: us-west-2
+discovery.seed_providers: ec2
```

Comment:
These are in the ami-elasticsearch.sh now.

Reply:
sorry, these stay here.  I forgot we still have these files.

Reply:
Got it!